theme-settings.php

```diff
@@ -17,18 +17,44 @@ function europa_form_system_theme_settings_alter(&$form, &$form_state) {
     ];
   }
 
-  $form['europa']['ec_europa_improved_website'] = [
+  $form['europa']['improved'] = [
+    '#type' => 'fieldset',
+    '#title' => t('Europa settings about improved websites'),
+  ];
+
+  $form['europa']['improved']['ec_europa_improved_website'] = [
     '#type' => 'checkbox',
     '#title' => t('Is this an "improved website"?'),
     '#description' => t('If this website is an "improved" one, we are going to show a site identification element in the page header.'),
     '#default_value' => theme_get_setting('ec_europa_improved_website', 'europa'),
+    '#weight' => 0,
+  ];
+
+  $form['europa']['improved']['ec_europa_improved_website_header'] = [
+    '#type' => 'radios',
+    '#title' => t('Will you use the "basic" page header or the "complete" one?'),
+    '#description' => t('You can choose between two different layouts for your page header, see <a href="@here">here</a>', ['@here' => 'https://ec-europa.github.io/ec-europa-theme/section-page-header.html']),
```

Comment:
Change the descrption to something more similar to this
`You can choose between two types of layout for your page header. For more details, please consult the <a href="@url">styleguide</a>`